stl/inc/chrono

```diff
@@ -60,7 +60,7 @@ namespace chrono {
     struct duration_values { // gets arithmetic properties of a type
         _NODISCARD static constexpr _Rep zero() noexcept {
             // get zero value
-            return _Rep(0);
+            return static_cast<_Rep>(0);

```

Comment:
This may be harmful, if Standard wants `Rep(0)` if it is supposed to work for fancy integers.

Reply:
What is the actual warning? Could we use `_Rep{0}`

Reply:
Warning suggest this form, actually

Reply:
https://docs.microsoft.com/en-us/cpp/code-quality/c26475?view=msvc-160

Reply:
I'm afraid there's a chance the change is observable with user-defined `Rep`, so reverted it.